src/nav/infrastructure.yml

```diff
@@ -457,11 +457,11 @@ pages:
             path: /docs/infrastructure/google-cloud-platform-integrations/troubleshooting/gcp-integration-api-authentication-errors
   - title: Microsoft Azure integrations
     pages:
-      - title: Introduction to Azure integrations
+      - title: Introduction to Azure integrations 
         path: /docs/infrastructure/microsoft-azure-integrations/get-started/introduction-azure-monitoring-integrations
-      - title: Azure Native New Relic Service 
+      - title: Azure Native New Relic 
         path: /docs/infrastructure/microsoft-azure-integrations/get-started/azure-native
-      - title: Activate Azure integrations
+      - title: Set up Azure integrations
```

Comment:
"Activate" feels like a more exciting verb... I'm curious why you switched it?

Reply:
Is there a way we can make it clear that this set up doc is where the action is, I guess is what I'm trying to get at?

It's kind of buried in the nav as it is

Reply:
the other cloud integrations didn't have 'activate' and had like 'connect' which I thought wasn't as good as 'set up'.